extensions/ql-vscode/gulpfile.ts/webpack.config.ts

```diff
@@ -55,6 +55,19 @@ export const config: webpack.Configuration = {
             loader: 'css-loader'
           }
         ]
+      },
+      {
+        test: /\.(woff(2)?|ttf|eot|svg)$/,
+        use: [
+          {
+            loader: 'file-loader',
+            options: {
+              name: '[name].[ext]',
+              outputPath: 'fonts/',
+              esModule: false
```

Comment:
Are we setting this to help with tree-shaking? 

It might be worth adding a code comment to explain - webpack config is a bit too magical.

Reply:
I've added a comment. If this is not set, Webpack doesn't know this is just a file path and will use `src: url([object Module]) format("truetype");` as the CSS property for the font.

Reply:
Cool, thanks for the explanation!